### ISSUE TITLE
Does the code still work ? 

### ISSUE THREAD

**Issue by ldb2000** [gboudreau/nest-api#86]
I have used this code a long time ago. But now I have lots of errors...

**Comment by bauzer714** [comment]
Yes it still works. Nest has made lots of changes and I use this class everyday for data aggregation.  Perhaps you need to update your class?

**Comment by ldb2000** [comment]
When I try the example.php I have the following error
Location information:
PHP Notice:  Undefined property: stdClass::$structure in /home/pi/nest-api/nest.class.php on line 159

**Comment by bauzer714** [comment]
Examples isn't really written to be executed in it's entirety.. it's simply to show you the things you can do with it.   However, I can confirm that he `$nest->getUserLocations();` works for me.  There isn't much being done in that function. Maybe on line 159 you can do a `var_dump($this->last_status); return;` and start tracing what's actually happening.
